src/Button/MeasureButton/MeasureButton.spec.jsx

```diff
@@ -0,0 +1,825 @@
+/*eslint-env jest*/
+
+import TestUtil from '../../Util/TestUtil';
+import Logger from '../../Util/Logger';
+
+import OlLayerVector from 'ol/layer/vector';
+import OlInteractionDraw from 'ol/interaction/draw';
+import OlFeature from 'ol/feature';
+import OlGeomLineString from 'ol/geom/linestring';
+import OlGeomPolygon from 'ol/geom/polygon';
+import OlOverlay from 'ol/overlay';
+import OlObservable from 'ol/observable';
+
+import { MeasureButton, MeasureUtil } from '../../index';
+
+describe('<MeasureButton />', () => {
+
+  let map;
+
+  beforeEach(() => {
+    map = TestUtil.createMap();
+  });
+
+  describe('#Basics', () => {
+
+    it('is defined', () => {
+      expect(MeasureButton).not.toBeUndefined();
+    });
+
+    it('can be rendered', () => {
+      const wrapper = TestUtil.mountComponent(MeasureButton, {
+        map: map,
+        measureType: 'line'
+      });
+      expect(wrapper).not.toBeUndefined();
+    });
+
+    it('measureType prop must have valid value', () => {
+      const wrapper = TestUtil.mountComponent(MeasureButton, {
+        map: map,
+        measureType: 'line'
+      });
+
+      const measureTypeValidValues = [
+        'line',
+        'area',
+        'angle'
+      ];
+
+      expect(measureTypeValidValues).toContain(wrapper.props().measureType);
+
+      let spy = {};
+      spy.console = jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
Is there any benefit of creating an object `spy` with property `console` instead of just creating a variable (can be const here btw) named `consoleSpy` or similar?